xAPI.md

```diff
@@ -2410,19 +2423,30 @@ identifier it describes.  We recommend that
 Activity Providers look for and use established, widely adopted identifiers for all types of IRI 
 identifiers other than Activity id.
 
-##### Requirements
+##### Metadata Provider Requirements
 
 * Metadata MAY be provided with an identifier.
 * If metadata is provided, both name and description SHOULD be included.
-* IRLs SHOULD be defined within a domain controlled by the person creating the IRL.
+* IRLs SHOULD be defined within a domain controlled by the [Metadata Provider](#def-metadata-provider) creating the IRL.
```

Comment:
Continuing on down, for that. IRL is a silly mistake (*my* silly mistake), and there's no problem switching them to IRI.